venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -13,24 +13,27 @@ function() {
     var BLIND_SUBMISSION = CONF + '/-/Blind_Submission';
 
     var withdrawProcess = `function() {
-      var or3client = lib.or3client;
-
-      var CONF = 'ICLR.cc/2018/Conference';
-      var BLIND_INVITATION = CONF + '/-/Blind_Submission';
-
-      or3client.or3request(or3client.notesUrl + '?id=' + note.referent, {}, 'GET', token)
-      .then(result => {
-        var blindedNote = result.notes[0];
-
-        var milliseconds = (new Date).getTime();
-        blindedNote.ddate = milliseconds
-        console.log('blindedNotes ' + JSON.stringify(blindedNote))
-        return blindedNote;
-      })
-      .then(blindedNote => or3client.or3request(or3client.notesUrl, blindedNote, 'POST', token))
-      .then(result => done())
-      .catch(error => done(error));
-      return true;
+        var or3client = lib.or3client;
+
+        var CONF = 'ICLR.cc/2018/Conference';
+        var BLIND_INVITATION = CONF + '/-/Blind_Submission';
+
+        or3client.or3request(or3client.notesUrl + '?id=' + note.referent, {}, 'GET', token)
+        .then(result => {
+            if (result.notes.length > 0){
+                var blindedNote = result.notes[0];
+
+                var milliseconds = (new Date).getTime();
+                blindedNote.ddate = milliseconds
+                return blindedNote;
+            } else {
+                console.log('No notes with the referent ' + note.referent + ' were found');
```

Comment:
you need to return something here, you can reject the promise with a message error:

```
else {
       console.log('No notes with the referent ' + note.referent + ' were found');
       return Promise.reject('No notes with the referent ' + note.referent + ' were found');
}
```